src/Components/Telemetry.md

```diff
@@ -257,6 +257,13 @@ Aspire.Npgsql.EntityFrameworkCore.PostgreSQL:
     - "db.client.connections.timeouts"
     - "db.client.connections.usage"
 
+Aspire.OpenAI:
+- Log categories:
```

Comment:
```suggestion
- Log categories:
  - none
```

Reply:
FYI - @annelo-msft who is working on the ILogger support for System.ClientModel.

Reply:
@m-redding is driving this work for SCM, we don't have a confirmed ship date for it yet, but I don't expect it to be a lot longer from here.